README.md

```diff
@@ -121,3 +121,44 @@ You can also pass nativefier flags, and mount additional volumes to provide loca
 ## License
 
 [MIT](LICENSE.md)
+
+## TS: Minimum to release beta
+
+- [x] Move cli to TS
+- [x] Move app to TS
+- [x] Package.json nits: move away from "^A.B.C" to more understandable "A.x
```

Comment:
I thought that the trend was to adopt the "semver" scheme with "major.minor.patch" kind of versions. Maybe this change will break expectations? I have started experimenting with this on a monorepo recently: https://github.com/remusao/mono/pulls?q=is%3Apr+is%3Aclosed the versioning can be automated based on pull requests labels (e.g. "Bug Fix" will increment 'patch', "New Feature" or "Enhancement" will increment 'minor' and "Breaking Change" will increment 'major'). I am using [auto](http://npmjs.com/package/auto) (alongside lerna, which is only needed in a monorepo setup) to automate this version bumping based on un-released pull requests and so far it has been great.

Reply:
> I'm a bit concerned that _[moving away from "^A.B.C" to more understandable "A.x"]_ cause dependency issues for users as we have a "looser" set of dependencies with potentially breaking changes

@jiahaog the move from `^A.B.C` to `A.x` doesn't change anything. I'm doing it because over the years I've seen a couple of people "bumping" e.g. `^3.6.3` to `^3.7.1`, which is pointless since both mean `3.x`. See  [about semantic versioning](https://docs.npmjs.com/about-semantic-versioning). It's almost purely equivalent, but one it the default and confusing, and one is obvious. ("almost" because `^3.6.3` means "3.x AND greater than 3.6.3", but for our needs we don't care). So, with this, less confusion. I'm for it.

> What's the best practice in the Node ecosystem these days? Should we use a lockfile instead?

@jiahaog yeah a lockfile is the best practice. But again, they're misunderstood by occasional maintainers, who are confused about how to update them, and if they don't then the build breaks, etc. Stability and respect for semver guarantees are quite good today in the npm ecosystem, so I don't feel the need for it. If that changes, we can add one.